slu-scripts/src/index.ts

```diff
@@ -64,9 +64,10 @@ export async function execScript(argv: any) {
 		}
 	} else if (argv._.includes('send-proof')) {
 		try {
-			const decryptedData = await decryptData(encryptedDataPath, keyFilePath);
 			if (interval) {
-				setInterval(() => sendAuthProof(decryptedData!, collectorBaseUrl), Number(interval));
+				setInterval(() =>
+					decryptAndSendProof(encryptedDataPath!, keyFilePath!, collectorBaseUrl!), Number(interval)
```

Comment:
shouldnt we use an await here?

Reply:
`setInterval(async() =>
					await decryptAndSendProof(encryptedDataPath!, keyFilePath!, collectorBaseUrl!), Number(interval)`